frontend/src/components/MapView/MapChat/index.js

```diff
@@ -108,66 +105,69 @@ class MapChat extends Component {
   }
 
   render = () => {
-    const rightOffset = this.state.open ? '0' : '-300px'
+    const rightOffset = this.props.chatOpen ? '0' : '-300px'
```

Comment:
is there a way to make the map chat hidden (display: none or just not rendered) and still render the junto button? Maybe restructure it a bit? because this bug is pretty annoying, you need to refresh the page

My theory is that you've got a form input in focus, you hit tab by accident or on purpose, and it jumps to the next form input (which is 300px offscreen in the map chat), and then the browser says "oh nooooo the cursor went offscreen" and tries its best to show the input, but then it screws up our css logic